sdk-base/src/main/java/com/mapbox/maps/module/MapTelemetry.java

```diff
@@ -1,52 +1,67 @@
-package com.mapbox.maps.module
+package com.mapbox.maps.module;
 
-import android.os.Bundle
+import android.os.Bundle;
+
+import androidx.annotation.Nullable;
+
+import com.mapbox.common.TelemetryUtils;
 
 /**
  * Definition of map telemetry
  */
-interface MapTelemetry {
+public interface MapTelemetry {
```

Comment:
Note that MapTelemetry interface is rewritten in Java: for more context in https://github.com/mapbox/mapbox-maps-android/pull/1670#discussion_r972473077, as we want to make the default interface backwards-compatible with Java, writing default method with `@JvmDefault` in Kotlin will require `-Xjvm-default compiler option`.

As summarised by @kmadsen 
> The -Xjvm-default compiler flag may do what we want in the future. But right now, it does not generate default interfaces in an SDK. The app developer has to include the flag in their app to make the app compile when it uses an sdk that is relying on the flag.